ago-dic-2018/Guadalupe Alejandro Pérez Camacho/practica_1/Alumno.java

```diff
@@ -0,0 +1,41 @@
+/*
+ * To change this license header, choose License Headers in Project Properties.
+ * To change this template file, choose Tools | Templates
+ * and open the template in the editor.
+ */
+package javaapplication5;
+
+/**
+ *
+ * @author Acer
+ */
+public class Alumno {
+    private String Nombre="";
+    private int contador;
+    Alumno(String n){
+        this.Nombre=n;
+        this.contador=1;
+    }
+
+    @Override
+    public String toString() {
+        return "Alumno{" + "Nombre=" + Nombre+'}';
```

Comment:
No estoy seguro, pero en estos métodos no tendrías que usar la misma sintaxis de `this.Nombre` ?

Reply:
Ayer lo estuve probando y funcionaba sin problemas, creo que más que nada el ***.this*** se usa de esta forma 
```
private String Nombre;
private int contador;
Alumno(String Nombre, int contador){
    this.Nombre=Nombre;
    this.contador=Contador;
}
```